src/commands/check/index.ts

```diff
@@ -111,20 +111,20 @@ export async function check(options: CheckOptions) {
       if (options.failOnOutdated)
         exitCode = 1
 
-      console.log(`Add ${c.green('-w')} to write package.json`)
+      console.log(`Add ${c.green('-w')} to write to package.json`)
     }
 
     console.log()
   }
   else if (hasChanges) {
     if (!options.install && !options.update && !options.interactive) {
       console.log(
-        c.yellow(`ℹ changes wrote to package.json, run ${c.cyan('npm i')} to install updates.`),
+        c.yellow(`ℹ changes written to package.json, run ${c.cyan('npm i')} to install updates.`),
```

Comment:
```suggestion
        c.yellow(`ℹ changes are written to package.json, run ${c.cyan('npm i')} to install updates.`),
```

This this a bit better?

Reply:
`changes written` is fine, but `changes are written` isn't - the changes have already been made, so this needs to be in the past tense, so either just `changes written` if you want to be terse, or `changes were written`.

Reply:
Oh yeah. I made the typo again haha.